### PR TITLE
feat(guides): complete some TODO guides

### PR DESCRIPTION
**Pull request by FC5570**

### CODE REVIEW

**Comment by kaname-png** [comment]
Tbh, The guide on how to create an argument needs more context, it lacks information.

File: docs/Guide/additional-information/using-and-extending-container.mdx [hunk 1]
```diff
@@ -3,4 +3,35 @@ sidebar_position: 1
...
+
+You can attach your custom properties to the container, as mentioned above, to access them in your code:
+
+```typescript ts2esm2cjs
+import { container } from '@sapphire/framework';
+
+container.property = 'value';
+
+console.log(container.property); // value
+```
```
[the remark this answers is not in the document]
Comment (FC5570) [reply docs/Guide/additional-information/using-and-extending-container.mdx:34]: Yes, forgot to do that. I will make changes

File: docs/Guide/arguments/creating-your-own-arguments.mdx [hunk 2]
```diff
@@ -3,4 +3,79 @@ sidebar_position: 2
 title: Creating your own arguments
 ---
 
-## TODO
+Creating an argument can be done by extending the Argument class imported by the framework:
```
[the remark this answers is not in the document]
Comment (FC5570) [reply docs/Guide/arguments/creating-your-own-arguments.mdx:6]: I mean to write, "Creating an argument can be done by extending the Argument class imported from the framework"

[the remark this answers is not in the document]
Comment (megatank58) [reply docs/Guide/arguments/creating-your-own-arguments.mdx:6]: > I mean to write, "Creating an argument can be done by extending the Argument class imported from the framework"

Maybe "Creating an argument can be done by extending the Argument class which can be imported from the framework"

**Review by favna — CHANGES_REQUESTED** [review]
Please rebase the branch to fix the vercel deploy. Main needed fixing after yesterday's framework PR merge.

**Review by favna — CHANGES_REQUESTED** [review]
Well done, you nuked your changed by using the app instead of doing it manually.

**Pull request merged**